test/wasmBenchmarker/ctests/mandelbrot.c

```diff
@@ -1,70 +1,64 @@
-/*
- * Copyright (c) 2023-present Samsung Electronics Co., Ltd
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
```

Comment:
Why did you remove these license comments?
I also wonder the origin of this test code.

Reply:
I accidently removed it. Thank you for noticing it.

When I started to write the SIMD Mandelbrot test, at first I made a non-SIMD version, then I made the SIMD version based on it. To ensure that both of the Mandelbrot tests calculate the same thing, I copied my non-SIMD version to the benchmark tests (replacing the old one). Now, their results can be compared.